### ISSUE TITLE
[BUG] Unsuccessful backup restore

### ISSUE THREAD

**Issue by vregat** [Paperback-iOS/app#568]
**Describe the bug**
Today I tried to restore backup to a clean installation of Paperback. After a while it showed parse errors and did not import bookmarked mangas.
The backup was created in the same version as it is being imported (latest version, of course)

**To Reproduce**
Create a larger backup (currently mine is over 10 MB, can provide)
Restore the backup to a clean installation

**Expected behaviour**
Import all mangas correctly

**App Version**
v0.7-r35

**Screenshots**
Attaching log instead
[04-05.Paperback.txt](https://github.com/Paperback-iOS/app/files/8618096/04-05.Paperback.txt)

**Comment by FaizanDurrani** [comment]
Errors are normal for now, they're because of the Paperback source; Delete that source and they should go away.

**Comment by vregat** [comment]
I thought it could be something like this. However I tried to clean install Paperback again, removed the preinstalled extension (no other sources, trackers, ...), imported the backup and the error is the same.

**Comment by FaizanDurrani** [comment]
Did you get the Paperback source after the backup finished importing? If so then it's likely the same issue.

**Comment by vregat** [comment]
No, did not get the Paperback source.
The source is not even in the original device where the backup was created.

**Comment by vregat** [comment]
So, yesterday I removed the app and cleaned all data. Installed the app again, removed the default source and add all my sources and mangas manually. After adding all the mangas I created a new backup and removed the app completely again.
Then I tried to recover from the backup (removed the default source again) and it still does not restore completely. The function restored 52 mangas out of 68.

Of course I installed the latest current version.
